### PR TITLE
chore: remove examples from repo

### PR DESCRIPTION
**Pull request by achingbrain**
js-libp2p examples now live in their own repo - https://github.com/libp2p/js-libp2p-examples

They are each replicated into their own repos, e.g. https://github.com/libp2p/js-libp2p-example-circuit-relay to make them easier to consume a la ipfs-examples/helia-examples.

## Change checklist

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation if necessary (this includes comments as well)
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,8 +31,8 @@
     "release": "run-s build docs:no-publish npm:release docs",
     "npm:release": "aegir exec --bail false npm -- publish",
     "release:rc": "aegir release-rc",
-    "docs": "NODE_OPTIONS=--max_old_space_size=8192 aegir docs -- --exclude interop --exclude examples/auto-relay --exclude examples/chat --exclude examples/connection-encryption --exclude examples/delegated-routing --exclude examples/discovery-mechanisms --exclude examples/echo --exclude examples/peer-and-content-routing --exclude examples/pnet --exclude examples/protocol-and-stream-muxing --exclude examples/pubsub --exclude examples/transports --exclude doc",
```
Comment (SgtPooki) [inline package.json:36]: wow

File: package.json [hunk 2]
```diff
@@ -33,8 +31,8 @@
...
-    "docs:no-publish": "NODE_OPTIONS=--max_old_space_size=8192 aegir docs --publish false -- --exclude interop --exclude examples/auto-relay --exclude examples/chat --exclude examples/connection-encryption --exclude examples/delegated-routing --exclude examples/discovery-mechanisms --exclude examples/echo --exclude examples/peer-and-content-routing --exclude examples/pnet --exclude examples/protocol-and-stream-muxing --exclude examples/pubsub --exclude examples/transports --exclude doc"
+    "docs": "NODE_OPTIONS=--max_old_space_size=8192 aegir docs -- --exclude interop --exclude doc",
```
Comment (SgtPooki) [inline package.json:34]: do we still need node_options with the reduction of all the examples?

File: doc/GETTING_STARTED.md [hunk 3]
```diff
@@ -276,7 +276,7 @@ localStorage.setItem('debug', 'libp2p:websockets,libp2p:webtransport,libp2p:kad-
 
 ## What is next
 
-There are a lot of other concepts within `libp2p`, that are not covered in this guide. For additional configuration options we recommend checking out the [Configuration Readme](./CONFIGURATION.md) and the [examples folder](../examples). If you have any problems getting started, or if anything isn't clear, please let us know by submitting an issue!
+There are a lot of other concepts within `libp2p`, that are not covered in this guide. For additional configuration options we recommend checking out the [Configuration Readme](./CONFIGURATION.md) and the [examples repo](https://github.com/libp2p/js-libp2p-examples). If you have any problems getting started, or if anything isn't clear, please let us know by submitting an issue!
```
Comment (SgtPooki) [inline doc/GETTING_STARTED.md:279]: could be in another PR, but "update us by opening a discussion" ?

File: README.md [hunk 4]
```diff
@@ -114,22 +114,11 @@ If you are starting your journey with `js-libp2p`, read the [GETTING\_STARTED.md
 
 ### Tutorials and Examples
 
-You can find multiple examples on the [examples folder](./examples) that will guide you through using libp2p for several scenarios.
+You can find multiple examples on the [examples repo](https://github.com/libp2p/js-libp2p-examples) that will guide you through using libp2p for several scenarios.
```
Comment (SgtPooki) [inline README.md:117]: i feel like there should be other tutorials we could link to here, but i cant think of any other than protoschool which needs updating

File: .github/workflows/examples.yml [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    runs-on: ubuntu-latest
+    needs: build
+    continue-on-error: true
+    strategy:
+      matrix:
+        example:
+          - name: js-libp2p-example-chat
+            repo: https://github.com/libp2p/js-libp2p-example-chat.git
+            deps:
+              - '@libp2p/peer-id-factory@$PWD/packages/peer-id-factory'
```
Comment (SgtPooki) [inline .github/workflows/examples.yml:33]: where is PWD coming from?

**Review by SgtPooki — APPROVED** [review]
looks good to me, and https://github.com/libp2p/js-libp2p-examples/actions/runs/6643582106 is passing. 

would love to see a diff of the new examples repo vs old examples folder

File: .github/workflows/examples.yml (hunk 5 above)
Comment (achingbrain) [reply .github/workflows/examples.yml:33]: From the shell, I think?

Comment (SgtPooki) [reply .github/workflows/examples.yml:33]: I didn't know you could do `@<path>` for npm versions...  TIL

File: README.md (hunk 4 above)
Comment (achingbrain) [reply README.md:117]: > Pours one out for Protoschool.

😢
